2024/README.md

```diff
@@ -108,7 +108,7 @@ All paper submissions that receive favorable reviews will be included as part of
 
 :------------------|:--------------------------------------
 ~~02 Aug 2024~~    | Paper submission deadline
-09 Aug 2024        | Paper submission deadline (extended)
+14 Aug 2024        | Paper submission deadline (extended)
```

Comment:
Maybe update the `(extended)` to `(second extension)` or `(extended 2x)`

Reply:
👍  I added the old date back in and put "second extension" on the new date